tilbakekreving/infrastructure/src/main/kotlin/tilbakekreving/infrastructure/client/TilbakekrevingSoapClient.kt

```diff
@@ -115,6 +118,74 @@ class TilbakekrevingSoapClient(
         }.flatten()
     }
 
+    /*
+    https://confluence.adeo.no/display/OKSY/Detaljer+om+de+enkelte+ID-koder?preview=/178067795/178067800/worddav1549728a4f1bb4ae0651e7017a7cae86.png
+     */
+    override fun annullerKravgrunnlag(
```

Comment:
Har vi en test på det å iverksette for klienten? Hvis ja, kan vi lage en for denne og?